trikot-kword/kword-remote-update/src/commonMain/kotlin/com/mirego/trikot/kword/remote/update/internal/RemoteTranslationsFetcher.kt

```diff
@@ -16,22 +17,27 @@ import kotlinx.serialization.json.Json
 internal class RemoteTranslationsFetcher(
     private val baseTranslationsUrl: String?,
     private val translationsVersion: String?,
-    private val internalCacheWrapper: InternalCacheWrapper
+    private val internalCacheWrapper: InternalCacheWrapper,
+    private val coroutineScope: CoroutineScope = CoroutineScope(Dispatchers.IO)
```

Comment:
You might want to use a supervisor scope for this one, otherwise, one failing http request might make other http requests fail.

See: https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/supervisor-scope.html

Reply:
Oui c'est vrai t'as bien raison! Je vais rouvrir une autre PR pour ca